src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -84,22 +89,31 @@ const SetAclPermsDocContainerForm = () => {
       return;
     }
 
-    try {
-      await setDocContainerAclPermission(session, permissions, podUrl, otherPodUsername);
+    // Routine for setting permissions to Documents
+    if (user === 'personal') {
```

Comment:
What are these changes?

Reply:
These are just to differentiate between the permission forms if they're in the profile page or the client profile page.